globalsearch/utils/check_utils.py

```diff
@@ -0,0 +1,59 @@
+import flask
+
+
+def check_dict_has_none_values(dictionary):
```

Comment:
Could be simplified as `return None in dictionary.values()`

Reply:
Yeah thanks! That is much better!